test/Experimental/MatrixGroupExperimental-test.jl

```diff
@@ -0,0 +1,10 @@
+@testset "GAP matrix group experiment" begin
+  _wrap_for_gap(m::MatrixElem) = GAP.Globals.MakeJuliaMatrixRep(m)
```

Comment:
```suggestion
   _wrap_for_gap(m::MatrixElem) = GAP.Globals.MakeJuliaMatrixRep(m)
```
Perhaps this should be in `experimental/Matrix/matrix.jl` as we want to use it from other code later on, too.